integration/tests/basic_functionality/negative/neg_deploy_with_missing_r10k_yaml.rb

```diff
@@ -9,7 +9,7 @@
 r10k_fqp = get_r10k_fqp(master)
 
 #Verification
-error_message_regex = /No configuration file given, no config file found in current directory, and no global config present/
+error_message_regex = /the supplied configuration does not define the 'sources' key/
```

Comment:
@adrienthebo Is there a reason that we cant say that we can't find a configuration file?  Since in this test we are specifically not supplying a configuration file.